src/e2e/specs/settings.spec.ts

```diff
@@ -40,18 +38,18 @@ test.describe(`${process.env.E2E_TEST_ENV} Settings Page`, () => {
 
     await expect(async () => {
       // select "send breach alerts to the affected email address" option
-      await settingsPage.sendToAffectedEmailRadioButton.click();
+      await settingsPage.sendToAffectedEmailRadioButton.click({ force: true });
 
       // verify option is selected
-      await expect(page.locator('input[data-alert-option="0"]')).toBeChecked();
-    }).toPass();
+      await expect(settingsPage.sendToAffectedEmailRadioButton).toBeChecked();
+    }).toPass({ timeout: 2000 });
 
     await expect(async () => {
       // select "send all breach alerts to the primary email address"
-      await settingsPage.sendToPrimaryEmailRadioButton.click();
+      await settingsPage.sendToPrimaryEmailRadioButton.click({ force: true });
```

Comment:
Possibly same comment as https://github.com/mozilla/blurts-server/pull/4054/files#r1462991417 above, re: adding comment explaining why `force: true` is necessary.